src/main/kotlin/org/rust/lang/core/psi/ext/RsItemElement.kt

```diff
@@ -23,3 +26,50 @@ fun <T : RsItemElement> Iterable<T>.filterInScope(scope: RsElement): List<T> {
     }
     return if (set.isEmpty()) toList() else toMutableList().apply { removeAll(set) }
 }
+
+val RsItemElement.itemKindName: String
+    get() = when (this) {
+        is RsMod, is RsModDeclItem -> "module"
+        is RsFunction -> "function"
+        is RsConstant -> when (kind) {
+            RsConstantKind.STATIC -> "static"
+            RsConstantKind.MUT_STATIC -> "static"
+            RsConstantKind.CONST -> "constant"
+        }
+        is RsStructItem -> when (kind) {
+            RsStructKind.STRUCT -> "struct"
+            RsStructKind.UNION -> "union"
+        }
+        is RsEnumItem -> "enum"
+        is RsTraitItem -> "trait"
+        is RsTraitAlias -> "trait alias"
+        is RsTypeAlias -> "type alias"
+        is RsImplItem -> "impl"
+        is RsUseItem -> "use item"
+        is RsForeignModItem -> "foreign module"
+        is RsExternCrateItem -> "extern crate"
+        is RsMacro2 -> "macro"
+        else -> "item"
+    }
+
+val RsItemElement.itemKindNameCapitalized: String
```

Comment:
Maybe it's better to use  `itemKindName.capitalize()` instead of new quite specific property?